### ISSUE TITLE
Filament does not use middleware group `web`

### ISSUE THREAD

**Issue by leo-unglaub** [filamentphp/filament#4398]
### Package

filament/filament

### Package Version

v2.16.22

### Laravel Version

v9.34

### PHP Version

PHP 8.1

### Problem description

All routes that belong to the admin panel are not put in the middleware group `web`. That's the reason the following default middlewares are not executed by default:

```
\App\Http\Middleware\EncryptCookies::class,
\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
\Illuminate\Session\Middleware\StartSession::class,
\Illuminate\View\Middleware\ShareErrorsFromSession::class,
\App\Http\Middleware\VerifyCsrfToken::class,
\Illuminate\Routing\Middleware\SubstituteBindings::class,
```

This causes an issue when people want to use a custom middleware to access the currently logged in user via `$request->user()` or `Auth::user()`. They always return `null` because `\Illuminate\Session\Middleware\StartSession` does not get executed.

### Expected behavior

All routes that are part of the admin panel should have the `web` middleware group assigned, because they behave like normal websites.

### Steps to reproduce

Register a custom middleware in `protected $middleware = [];` and try to access `Auth::user()`.

### Reproduction repository

https://github.com/filamentphp/filament/

**Comment by danharrin** [comment]
This is deliberate, we have our own middleware stack in the Filament config file

**Comment by leo-unglaub** [comment]
> This is deliberate, we have our own middleware stack in the Filament config file

Is it possible to inject something in that stack?

**Comment by leo-unglaub** [comment]
So the best way to extend this is to export the config file and then extend the list? Or is there a better way of doing that?

**Comment by danharrin** [comment]
Yeah, because not everyone wants app middleware to be applied to their admin panel

**Comment by dpenacton** [comment]
🛠 **Trouble with Export/Import Functionality in Laravel Filament Admin Panel: 'Login Route Not Found' Error**

I've been tackling the export/import feature, aiming for smooth functionality. However, upon attempting to export or import a CSV, it seems to generate within the notifications. Yet, when I endeavor to download the file, a persistent 'login route not found' error arises.

![Screenshot](https://img.enacton.com/ShareX/2024/04/w4FMvKPUQJ.png)

I've configured separate guards for authentication, catering to both admin and user panels. Additionally, I've ensured that the export/import policy is correctly set to return true within the view function.

Furthermore, I've diligently included `Export::polymorphicUserRelationship();` in the app service provider and executed all requisite database changes.

Despite these efforts, I find myself still grappling with the 'login route not found' issue when attempting to access export/import. Any suggestions on resolving this within the Laravel Filament admin panel?
